http-servers/src/main/java/com/palantir/remoting1/servers/TraceIdLoggingFilter.java

```diff
@@ -39,16 +40,32 @@ public void init(FilterConfig filterConfig) throws ServletException {}
     @Override
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
             throws IOException, ServletException {
+        try (Closeable ignored = populateTraceContext(request)) {
```

Comment:
Went with more explicit try/finally. 

Was previously `ignored` as Idea doesn't warn on unused var in try-with-resources (though it is implicitly used & closed).